test/files/run/repl-completions.scala

```diff
@@ -0,0 +1,11 @@
+import scala.tools.partest.ReplTest
+
+object Test extends ReplTest {
+  def code = """
+    |// completions!
+    |:completions "foo".com
+    |val someName = "foo"
+    |:completions someN
+    |:completions "foo".substring(1).rep
```

Comment:
Could you please modify the test to only consider completions from a class defined in this REPL session? It would then be less fragile when new methods matching "rep*" or "com*" are added to the standard library types used.

Reply:
See https://github.com/scala/scala/blob/9291e12004d6c96121467b1468249cb04afede2e/test/junit/scala/tools/nsc/interpreter/CompletionTest.scala#L50 for inspiration.

Reply:
will do